### PR TITLE
Chat view padding issue removed

### PR DESCRIPTION
**Pull request by muhammadusama95**
### Details
Implemented Solution is:
Go to ReportActionCompose.js File. Go to Parent View then there are multiple containers i.e. attachments, send button, and input text field. There was a problem with the input text field view styles. There wad Unnecessary padding of the input text field which was causing the extra space at the top and was not setting the input text view to the center of the parent view So we removed that and input text is at the center of the parent view and the top spacing has been removed using top, bottom padding value to 0 and I have tested this.
I am waiting to add me to the git link SO I can push the code to my branch and your team can review the code.

### Fixed Issues
Remove Top Padding from chatView.

### Tests
Tested on Android OnePlus 3 Mobile.
Tested on iOS iPhone X emulator.

### Tested On
- [x] iOS
- [x] Android

#### iOS
iOS_ScreenShot

#### Android
android_screenshot

### CODE REVIEW

**Comment by roryabraham** [comment]
Hi @muhammadusama95 - just a few early comments on this PR:

1) Please include the link to the issue you've been assigned under the **Fixed Issue** header
2) Please be sure to test any and all changes on all platforms, including screenshots (web, desktop, mobile web, iOS, Android). Right now, I only see iOS and Android
3) Looking at the screenshot you provided on Android, the text in the input field is vertically misaligned

**Comment by muhammadusama95** [comment]
Hi, @roryabraham Thanks for your review of my code.
I just made a new commit, Added Comments on code where I changed for the fixing of the top padding issue. So you can easily
identify my code.
Below are my answers to all your questions:
1: https://github.com/Expensify/Expensify.cash/issues/1002

2: Yeah As per my discussion with Michael, This is a cross-platform app So I have tested it on both iOS and Android. Not included web, desktop, or mobile web version.

3: By mistakenly added an old android screenshot Please see attached updated android screenshot too.
UpdatedAndroidScreenshot

4: Now I added comments in the new commit So you can quickly identify my changes.

5: There is no new lint error or warnings in my branch as I just changed the values of padding.

**Review by roryabraham — CHANGES_REQUESTED** [review]
Hi @muhammadusama95, sorry if I wasn't clear before, but I have a few new comments.

1) All the code style changes you have made to `src/styles/styles.js` are unacceptable, because they do not conform to our code style guidelines and the code format throughout the rest of the codebase. Adding comments to identify your code is not a valid solution - the comments would litter the codebase with irrelevant information. So to be clear, we will not merge any PR containing extraneous changes not relevant to the issue which you have been assigned. Why did you remove all indentation from the file?
2) _All_ PRs must be tested on 5 platforms: web, mobile web, desktop, android, and iOS. The same code is used to build the application on all 5 platforms, so we need to make sure your changes don't cause undesirable effects in the other platforms.
3) It seems like the screenshots you've provided render only the `ReportActionCompose` component in isolation, not the complete application.
4) I noticed that your code contains a platform-check, like so:

```
if (Platform.OS == 'ios') {
    ...
}
```

This is something we have decided to intentionally avoid in this codebase. Instead, check out the [README section covering platform-specific file extensions](https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions).

**Comment by muhammadusama95** [comment]
Hi @roryabraham,
Please check the last 3 commits So you can check the indentation Set, Comments removed, and padding issue fixed.

Now I have tested it using login through Expensify app and check below screenshots of both iOS and Android respectively.
iOS_ScreenShot_updated
Android_ScreenShot_updated

About testing it on All Platforms @michaelhaxhiu had chat with me on android and iOS only. Check Below Screenshot.
Michael_ScreenShot

**Comment by roryabraham** [comment]
Hi @muhammadusama95. Regardless of what others may have told you, _all PRs must be tested on all platforms_. So that means that your code changes must be tested on:

1) Web (from a laptop)
2) Desktop (from a mac computer - let me know if you don't have a mac)
3) Web (from a mobile device or simulator)
4) iOS
5) Android

**Review by roryabraham — CHANGES_REQUESTED** [review]
The diff is more reasonable now, but please review the diff and remove all non-critical changes.

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:11]: This indentation change is unnecessary, please revert

File: src/pages/home/report/ReportActionCompose.js
Comment (roryabraham) [inline src/pages/home/report/ReportActionCompose.js:178]: Please remove this blank line

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:43]: Please remove this blank line

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:47]: Please remove this blank line

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:64]: Please remove this blank line

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:330]: Please remove this blank line

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:308]: This indentation change is unnecessary, please revert

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:391]: Please indent this comment four spaces

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:539]: This indentation change is unnecessary, please revert

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:607]: This indentation change is unnecessary, please revert

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:643]: The change to `fontWeight` here shouldn't be necessary - please use `fontWeightBold` instead

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:991]: This indentation change is unnecessary, please revert

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:1024]: This indentation change is unnecessary, please revert

**Review by roryabraham — CHANGES_REQUESTED** [review]

**Comment by muhammadusama95** [comment]
Hi @roryabraham,
Please find All Platform tested screenshots.
1: Android
Android_ScreenShot_updated
2:iOS
iOS_ScreenShot_updated
3:Mobile Web
mobile_web
4:Web
web_
5:MAC-Desktop
mac_desktop

**Comment by roryabraham** [comment]
Looks like you've got a merge conflict

File: src/pages/home/report/ReportActionCompose.js
Comment (roryabraham) [inline src/pages/home/report/ReportActionCompose.js:221]: Please remove this unnecessary whitespace change

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:40]: Please don't change the indentation utils here. If you need `marginTop: 20`, use `mt5`

Comment (roryabraham) [reply src/styles/styles.js:40]: Also please make sure that these are arranged sequentially like `mt1, mt2, mt3...`

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:49]: Why did you remove these three styles?

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:71]: Why did you remove these two styles?

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:98]: Why did you remove this style?

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:204]: Why did you remove the fontFamily?

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:178]: Why did you remove these styles?

File: src/styles/styles.js
Comment (roryabraham) [inline src/styles/styles.js:989]: These comments need to be indented by one space

**Review by roryabraham — CHANGES_REQUESTED** [review]

**Comment by michaelhaxhiu** [comment]
Let's give @roryabraham a chance to review, as he's been doing 🔧

**Comment by roryabraham** [comment]
@muhammadusama95 There are still lots of unresolved comments that would need to be addressed before this could be merged. Not to mention there's a merge conflict, so it's not even possible to merge presently.

**Comment by muhammadusama95** [comment]
@michaelhaxhiu @roryabraham Can you please assign this issue to @ahmad.zahid2000@gmail.com. I will create a new branch and take a pull request instead of resolving these comments by doing this it's hardly 20-25 minutes work.

**Comment by muhammadusama95** [comment]
On the new branch, you will get only the required changes. Nothing else.

**Comment by michaelhaxhiu** [comment]
@muhammadusama95 We have decided to move forward with another contributor. Thanks for your assistance thus far, but no further action is required from your side.

I'm closing this PR and going to pay you in Upwork shortly.

**Pull request closed without merging**
